docs/LibraryEvolution.rst

```diff
@@ -787,9 +800,8 @@ clients that the enum cases are exhaustive. In particular:
 
 - Adding new cases is not permitted.
 - Reordering existing cases is not permitted.
-- Adding a raw type to an enum that does not have one is not permitted -- it's
-  used for optimization.
 - Removing a non-public case is not applicable.
+- Adding a raw type is still permitted.
```

Comment:
Are you sure about this?

Reply:
Perhaps @DougGregor or @jckarter knows if adding a raw type has any effects in addition to what you'd get if you wrote the conformance manually?

Reply:
I feel like I remember some from SE-0192 but perhaps I didn't actually implement them. (I know at one point we tossed around the idea of raw type = trivial one-word or 64-bit representation.)

Reply:
A raw type is just sugar for implementing a RawRepresentable conformance. It doesn't change the representation.

Reply:
Okay.